ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarkDetailsView.swift

```diff
@@ -73,6 +78,11 @@ private class URLTextField: UITextField {
       super.text = suggestedText
     }
   }
+
+  @objc
+  private func editingChanged() {
+    oldText = super.text
```

Comment:
How come this is needed when you already have a custom setter on `text`? 🤔

Reply:
The user can edit the text and when you press the save button, it doesn't update the internal text. So this is needed to update it. Otherwise we can modify the `save` button so that it dismisses the keyboard first, then saves. That's because once the keyboard is dismissed, only then is the text committed.

Reply:
Maybe toss a comment in here explaining that 🙂